Set Apache to allow unlimited requests per TCP connection.

jeremycline opened the pull request:
By default, Apache allows 100 HTTP requests per TCP connection before
resetting it. Since it is not uncommon for Pulp to serve thousands or
tens of thousands of requests from a single client, it makes sense to
not use the default. This is especially noticeable when performing
Pulp-to-Pulp syncs.

This is related to issue #1629.

mhrivnak commented:
Please bring this directly to katello's attention. This is the kind of change they could easily overlook.

jeremycline commented:
@mhrivnak FYI I updated the default to be 10,000 (admittedly still quite large) rather than unlimited, and added an inline comment about tweaking it based on DoS concerns.
